spec/support/helpers/features_helper.rb

```diff
@@ -2,15 +2,15 @@
 
 module FeaturesHelper
   def signin_spo_user(name = 'MOIC_POM')
-    mock_sso_response(name, ['ROLE_ALLOC_MGR'])
+    mock_sso_response(name, [SsoIdentity::SPO_ROLE])
   end
 
   def signin_global_admin_user
```

Comment:
The same thing applies here about dropping 'global' from the name.

```suggestion
  def signin_admin_user
```

I think we can safely refer to this type of user as an "admin user".